### DISCUSSION TITLE
アドベントカレンダー上からの参加記事本体へのジャンプ方法など

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#599] [要望・アイデア・不具合]
Qiita のアドベントカレンダーでは、**他のサイトへの投稿記事**も参加対象にしているため、多くの情報に接することができ、知見を広めるのに大いに役立っています。しかし、投稿者の意向次第で参加方法がマチマチです。

+ **参加方法１**　Qiita の記事上で、他のサイト上の記事の簡単な紹介をしたうえで、その記事にジャンプできるようなリンクを設けている。
+ **参加方法２**　Qiita には何の記事もなく、他のサイトの記事をそのまま登録している。

**参加方法１**の場合、カレンダー上の記事タイトルをクリックしたときの挙動が、去年と今年で違うような気がします（記憶は曖昧）。昨年は一旦 Qiita の記事に飛び、紹介されたリンクを閲覧者がクリックして最終記事に飛ぶような仕様だったように思います。しかし、今年は一気に最終記事に飛ぶようになりました。これでは、Qiita の記事の存在には気付きません。その結果、「いいね」も付けられないし、後日参考にするための「ストック」もできません。**まずは Qiita の記事に飛ぶような仕様の方が好ましく感じます**。

**参加方法２**の場合、当然、他のサイトに直接飛んでしまうため、Qiita だけの会員資格では記事に対して何のリアクションもできません。Qiita のカレンダーに参加している以上、投稿者は Qiita 会員でもあることは確かなので、**Qiita 上に紹介記事も書くことを参加条件**にしては如何でしょうか？　投稿者が「いいね」などは期待しないと考えているにしても、Qiita 閲覧者が「ストック」もできないようでは不便です。

**Comment by tanaka-asami415** [comment]
@tsubolabo 
ご意見ありがとうございます🙏

> 参加方法１の場合、カレンダー上の記事タイトルをクリックしたときの挙動が、去年と今年で違うような気がします

カレンダー上から記事へ移動する際の挙動は仕様変更をしていないため、2022年と2023年では同様の挙動となっています。
そのため、カレンダーからのリンク先は各ユーザーさんが指定したものとなっています。

> Qiita 上に紹介記事も書くことを参加条件にして

他サイトに誘導することを目的とした記事の場合、Qiitaの利用規約・コミュニティガイドライン違反に該当するため参加条件にすることは難しいです。

しかしながら、「Advent Calendar のQiita外の記事であっても、後から読み返しをしやすくする」といった方向で、
閲覧者体験を改善することは検討したいと思います🙇

**Reply by tsubolabo** [reply]
@tanaka-asami415 
> カレンダー上から記事へ移動する際の挙動は仕様変更をしていないため、2022年と2023年では同様の挙動となっています。

確かに、MATLAB/Simulink Advent Calendarで、部分的に再確認したところ次のとおりでした。昨年は、たまたまQiitaを経由して飛べるようになっていた記事もあったということのようです。

>2022_シリーズ1/2_5日目　
　Qiitaの記事があるのに、カレンダーのクリックから直接noteへ。
2022_シリーズ2/2_18日目
　カレンダーのクリックで一旦Qiitaの記事へ。そこからnoteへリンク。
2023_シリーズ1/2_5日目
　Qiitaの記事があるのに、カレンダーのクリックから直接noteへ。
2023_シリーズ2/2_10日目
　Qiitaの記事があるのに、カレンダーのクリックから直接noteへ。


> カレンダーからのリンク先は各ユーザーさんが指定したもの

確かに、そのようです。とは言っても、閲覧者からすると、色々なモードが存在すると混乱します。
強要するわけではありませんが、Qiitaの関連記事も存在する場合には、投稿手順の説明文に少々手を加えることでモードの混在は改善できそうに思えます。

> 他サイトに誘導することを目的とした記事の場合、Qiitaの利用規約・コミュニティガイドライン違反に該当

とすると、「2022_シリーズ2/2_18日目」の投稿方法は「ガイドライン違反」になるのですか？
タイトルをクリックしただけで、見ず知らずの他のサイトにいきなり飛んでしまう方が問題が多いような気がしますが・・・。

「アドベントカレンダーにはどのような記事を紐付けていいのですか？」というＱＡには次のような説明があります。「Qiita記事の紐付け」を「ガイドライン違反」にならないように行うのは難しそうですね。

>Qiita Advent CalendarにはQiitaはもちろんZennやnote、はてなブログ、個人ブログなど、エンジニアに関わる内容であればどのサイトの記事でも歓迎です。
ただしランキングへの反映や一部のイベント参加にはQiita記事の紐付けが必要となりますのでご注意ください。

私にとって、本件は「テーブルや数式の表示不具合」に相当するような重要な問題ではありません。なんとなく整合性がとれない複雑な話なので、頭の整理のために書いてみました。強要することは一つもありません。ご参考まで。

**Reply by tanaka-asami415** [reply]
@tsubolabo 
個別の記事についてDiscussions上で言及することは控えさせていただきますが、違反かどうかは、原則としてユーザーさまからの通報を元に当社で検討・判断をしておりますので、
記事に外部サービスのURLしかないような状況でかつ通報が多くあった場合、ガイドライン違反に該当すると判断される可能性が高いと言えます。

しかしながら、Advent Calendarでは他サービスの記事でのご参加もOKとしており、詳細な条件も設けていないこともありますので、
あまり杓子定規に判断せず、通常時の投稿よりは大目に見ていただいているのが実状ではないかと思います。

運営としても厳しく取り締まることよりも、イベントを楽しんでいただいたり、知見が多くの方に共有されることを望んでおります。
Qiitaとしては、Qiitaに記事を書いていただけるのが一番うれしいのですが、参加されるユーザーさまのご都合もあると思いますので
可能な限りユーザーのみなさまにお任せしている状態となります。

> なんとなく整合性がとれない複雑な話なので、頭の整理のために書いてみました。強要することは一つもありません。ご参考まで。

購読中のカレンダーに投稿された記事をフィード形式で一覧できる [カレンダーフィード](https://qiita.com/advent-calendar/2023/article-feed) では、
Qiitaの記事かそれ以外のリンクかが分かるようマーク表示をしているのですが、カレンダーの方にも何かしら違いが認識できるような改善は検討したいと思います🙇
